cmd/client.go

```diff
@@ -0,0 +1,64 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/thegreenwebfoundation/grid-intensity-go/pkg/provider"
+)
+
+func getClient(providerName string, cacheFile string) (provider.Interface, error) {
```

Comment:
This is used by both the root and exporter commands to set up a client for the relevant provider.